package.json

```diff
@@ -11,7 +11,23 @@
     "lint": "eslint .",
     "lint:fix": "eslint --fix .",
     "stylelint": "stylelint **/*.css",
-    "stylelint:fix": "stylelint --fix **/*.css"
+    "stylelint:fix": "stylelint --fix **/*.css",
+    "prepare": "husky install"
```

Comment:
Add npm script `"precommit": "lint-staged",`

Reply:
Have this as reminder to add it once jest is configured:

`"prepush": "jest --bail --changedSince=origin/main --coverage --maxWorkers=50%",`